### PR TITLE
[All hosts](samples) Add new samples

### PR DESCRIPTION
**Pull request by davidchesnut**
Updates the sample list to match latest changes in the repo.

### CODE REVIEW

File: docs/overview/office-add-in-code-samples.md [hunk 1]
```diff
@@ -18,16 +18,13 @@ The following samples show how to build the simplest Office Add-in with only a m
...
 
-<br>
-
----
-
----
-
 ## Blazor WebAssembly
 
+If your development background is in building VSTO Add-ins, the following samples show how to build Office Web Add-ins using .NET Blazor Webassembly. You can keep much of your code in C# and Visual Studio.
```
Comment (samantharamon) [inline docs/overview/office-add-in-code-samples.md:23]:
```suggestion
If your development background is in building VSTO Add-ins, the following samples show how to build Office Web Add-ins using .NET Blazor WebAssembly. You can keep much of your code in C# and Visual Studio.
```

File: docs/overview/office-add-in-code-samples.md [hunk 2]
```diff
@@ -74,6 +65,12 @@ The following samples show how to build the simplest Office Add-in with only a m
 | [Single Sign-on (SSO) Office Add-in with ASP.NET](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Office-Add-in-ASPNET-SSO) | Use the `getAccessToken` API in Office.js to give the add-in access to Microsoft Graph data. This sample is built on ASP.NET. |
 | [Single Sign-on (SSO) Office Add-in with Node.js](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Office-Add-in-NodeJS-SSO) | Use the `getAccessToken` API in Office.js to give the add-in access to Microsoft Graph data. This sample is built on Node.js.|
 
+## Office
+
+| Name                | Description         |
+|:--------------------|:--------------------|
+| [Save custom settings in your Office Add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/office-add-in-save-custom-settings) | Save custom settings inside an Office Add-in. The add-in stores data as key/value pairs, using the JavaScript API for Office property bag, browser cookies, web storage (localStorage and sessionStorage), or by storing the data in a hidden div in the document. |
```
Comment (samantharamon) [inline docs/overview/office-add-in-code-samples.md:72]:
```suggestion
| [Save custom settings in your Office Add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/office-add-in-save-custom-settings) | Save custom settings inside an Office Add-in. The add-in stores data as key-value pairs, using the JavaScript API for Office property bag, browser cookies, web storage (localStorage and sessionStorage), or by storing the data in a hidden div in the document. |
```

**Review by samantharamon — APPROVED** [review]
Added a couple of minor suggestions, but good to go otherwise.

**Review by AlexJerabek — APPROVED** [review]
Approving and merging on behalf of @davidchesnut, since I made a similar change.

**Pull request merged**